src/main/java/co/cask/plugin/MultiTableDBSource.java

```diff
@@ -71,6 +73,13 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
                       conf.getJdbcPluginName()));
     }
     pipelineConfigurer.getStageConfigurer().setOutputSchema(null);
+    try {
+      if (conf.getDateFormat() != null) {
```

Comment:
should also check if its not a macro. `!conf.containsMacro("dateFormat") && !Strings.isNullOrEmpty(conf.getDateFormat())`

Reply:
Fixed.